packages/react-examples/src/react/ComboBox/ComboBox.Inline.Example.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { ComboBox, IComboBoxOption, IComboBoxStyles, SelectableOptionMenuItemType } from '@fluentui/react';
+
+const options: IComboBoxOption[] = [
+  { key: 'Header1', text: 'First heading', itemType: SelectableOptionMenuItemType.Header },
+  { key: 'A', text: 'Option A' },
+  { key: 'B', text: 'Option B' },
+  { key: 'C', text: 'Option C' },
+  { key: 'D', text: 'Option D' },
+  { key: 'divider', text: '-', itemType: SelectableOptionMenuItemType.Divider },
+  { key: 'Header2', text: 'Second heading', itemType: SelectableOptionMenuItemType.Header },
+  { key: 'E', text: 'Option E' },
+  { key: 'F', text: 'Option F', disabled: true },
+  { key: 'G', text: 'Option G' },
+  { key: 'H', text: 'Option H' },
+  { key: 'I', text: 'Option I' },
+  { key: 'J', text: 'Option J' },
+];
+// Optional styling to make the example look nicer
+const comboBoxStyles: Partial<IComboBoxStyles> = { root: { maxWidth: 300 } };
+
+export const ComboBoxInlineExample: React.FunctionComponent = () => {
+  return (
+    <div>
```

Comment:
You might want to give this div a certain height because it clips as it renders right now:

![image](https://user-images.githubusercontent.com/7798177/125868566-f099ba23-86cb-413c-bac8-d58dc84f9737.png)

Reply:
Or maybe it needs a z-index or something? Also seems like the lack of layering is messing up the positioning logic, if it's trying to open up rather than down.

Reply:
it's because all the examples are in `overflow: hidden` containers, and it has no space below it 😅. Fixed now!